src/cli.ts

```diff
@@ -209,9 +213,16 @@ const createPatientBundles = (patientBundleDir: string) => {
  * Measure Reports to file.
  */
 const runMeasureCalculation = async () => {
+  if (options.from !== undefined && !moment(options.from).isValid())
+    console.log(
+      'Date format of --from param : ' + options.from + ' is not valid at ' + moment(options.from).invalidAt()
+    );
+  if (options.to !== undefined && !moment(options.to).isValid())
+    console.log('Date format of --to param : ' + options.to + ' is not valid at ' + moment(options.to).invalidAt());
+
```

Comment:
```suggestion
  if (options.from !== undefined && !moment(options.from).isValid())
   throw new Error(
      'Date format of --from param : ' + options.from + ' is not valid at ' + moment(options.from).invalidAt()
    );
  if (options.to !== undefined && !moment(options.to).isValid())
    throw new Error('Date format of --to param : ' + options.to + ' is not valid at ' + moment(options.to).invalidAt());

```
I was able to successfully get these messages logged to the console using the examples you provided - thank you! 👍 
Since there are many different items being logged to the console throughout the export, it might be hard to locate these messages if they do occur. I suggest changing the messages from `console.log` to `throw new Error` so that the messages are more explicit and so that they do not get passed to the measure calculation engine if they are invalid. Right now they still get passed to the engine since an error does not get thrown.

Reply:
I wondered if that part should be handled by just warning or hard stop before execution. If the latter I like the idea of throwing error, as users won't get good results anyway in the end.